library/ecjpake.c

```diff
@@ -1059,6 +1063,7 @@ int mbedtls_ecjpake_self_test( int verbose )
     if( verbose != 0 )
         mbedtls_printf( "passed\n" );
 
+#if !defined(MBEDTLS_ECJPAKE_ALT)
```

Comment:
Please add a comment explaining why those tests are not relevant in the case of an alternative implementation of ECJPAKE.

Reply:
Done.